qcodes/instrument_drivers/QuantumDesign/DynaCoolPPMS/DynaCool.py

```diff
@@ -0,0 +1,79 @@
+from qcodes.instrument.visa import VisaInstrument
+from functools import partial
+import logging
+
+import qcodes.utils.validators as vals
+
+log = logging.getLogger(__name__)
+
+
+class DynaCool(VisaInstrument):
+    """
+    Class to represent the DynaCoolPPMS
+
+    Note that this driver assumes the server.py (from the 'private' folder)
+    to be running on the DynaCool dedicated control PC.
+
+    Args:
+        name: The name used internaly by QCoDeS for this driver
+        address: The VISA ressource name.
+          E.g. 'TCPIP0::127.0.0.1::5000::SOCKET' with the appropriate IP
+          address instead of 127.0.0.1
+    """
+
+    def __init__(self, name: str, address: str, **kwargs) -> None:
+        super().__init__(name=name, address=address, terminator='\r\n',
+                         **kwargs)
+
+        self.add_parameter('temperature',
+                           label='Temperature',
+                           unit='K',
+                           get_parser=partial(self._pick_one, 1, float),
+                           get_cmd='TEMP?')
+
+        self.add_parameter('temperature_setpoint',
+                           label='Temperature setpoint',
+                           unit='K',
+                           vals=vals.Numbers(1.8, 400),
+                           set_cmd=partial(self._temp_setter,
+                                           'temperature_setpoint'))
+
+        self.add_parameter('temperature_rate',
+                           label='Temperature settle rate',
+                           unit='K/s',
+                           vals=vals.Numbers(0.0002, 0.3),
+                           set_parser=lambda x: x*60,  # conversion to K/min
```

Comment:
is this really more convenient than the instrument's default, which is K/min (I assume)? I am worried that users may get confused when using the driver while reading the manual of the instrument :)

Reply:
QCoDeS uses SI units. That is a simple rule to avoid confusion.